recipients_app/lib/core/cubits/auth/auth_cubit.dart

```diff
@@ -23,12 +23,14 @@ class AuthCubit extends Cubit<AuthState> {
       if (user != null) {
         try {
           final recipient = await userRepository.fetchRecipient(user);
+          final Organization? organization = await _fetchOrganization(recipient);
```

Comment:
You can always just write
final organization = await ....;
instead of 
final Organization? organization = await ....;

Type will be assigned automatically. Just FYI :D